Makefile

```
@@ -42,8 +42,12 @@ docker_build_front:
 docker_up:
 	docker-compose up -d
 
+.PHONY: docker_up_local
+docker_up_local:
+	docker-compose up -d -f docker-compose-local.yml
```

```suggestion
	docker-compose  -f docker-compose-local.yml up -d
```
I seem to get the following error
```sh
docker-compose up -d -f docker-compose-local.yml
unknown shorthand flag: 'f' in -f
```